acme_dns_tiny.py

```diff
@@ -0,0 +1,402 @@
+#!/usr/bin/env python3
+# pylint: disable=multiple-imports
+"""ACME client to met DNS challenge and receive TLS certificate"""
+import argparse, base64, binascii, configparser, copy, hashlib, ipaddress, json, logging
+import re, sys, subprocess, time
+import requests
+import dns.exception, dns.query, dns.name, dns.resolver, dns.rrset, dns.tsigkeyring, dns.update
```

Comment:
Does this work for you out of the box?

Quickly testing this on ESXi-7.0U2d with the installed Python 3.8.3 tells me:  
`ModuleNotFoundError: No module named 'dns'` which makes to me as https://www.dnspython.org is not shipped.